website/src/lib/wordpress/index.js

```diff
@@ -156,45 +132,34 @@ export async function getNodeType(slug) {
 /**
  * Get fields for single page regardless of post type.
  */
-export async function getContent(slug, preview, previewData) {
+export async function getContent(
+  slug,
+  preview,
+  previewData,
+  options,
```

Comment:
I was getting `TypeError: Cannot set properties of undefined (setting 'uriType')` down on line 158. Changing this to `options = {}` fixed. Now what I'm not sure of is if we actually want to pass in an options object as a variable here, or simply define inside the function.